.github/workflows/install-index-release.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches: [ "index-on-ci" ] # FIXME
```

Comment:
Did you intend to change this before merging?

Reply:
Yup, I'll drop the commit before merging